zerver/tests/test_hooks.py

```diff
@@ -1181,3 +1181,13 @@ def test_circleci_build_in_success_status_when_previous_build_failed_too(self):
         expected_subject = u"RepoName"
         expected_message = u"[Build](https://circleci.com/gh/username/project/build_number) triggered by username on master branch fixed."
         self.send_and_test_stream_message('build_passed_when_previous_build_failed', expected_subject, expected_message)
+
+class CrashlyticsHookTests(WebhookTestCase):
+    STREAM_NAME = 'crashlytics'
+    URL_TEMPLATE = "/api/v1/external/crashlytics?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'crashlytics'
+
+    def test_circleci_build_in_success_status(self):
+        expected_subject = u"Issue report"
```

Comment:
We should have some unique identifier for the issue as part of the subject.  I think something like this would be pretty reasonable:

u'{}: {}'.format(display_id, title)